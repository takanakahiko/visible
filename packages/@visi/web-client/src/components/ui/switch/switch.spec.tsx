import { render } from '@testing-library/react';
import React from 'react';

import { Switch } from '.';

describe('Switch', () => {
  it('matches snapshot', () => {
    const { container } = render(<Switch>test</Switch>);
    expect(container.firstChild).toMatchSnapshot();
  });
});
