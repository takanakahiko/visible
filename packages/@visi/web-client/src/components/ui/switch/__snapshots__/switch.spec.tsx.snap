// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Switch matches snapshot 1`] = `
<label
  class="inline-flex"
>
  <button
    aria-checked="false"
    class="rounded-full h-6 w-10 shadow-inner mr-2 relative transition-colors ease-in-out duration-150 outline-none focus:outline-none focus:shadow-outline bg-gray-500"
    role="switch"
  >
    <div
      aria-hidden="true"
      class="rounded-full bg-white w-4 h-4 shadow-md absolute top-0 bottom-0 my-auto mx-1 transition duration-150 ease-in-out right-auto"
    />
  </button>
  <div
    class="select-none"
  >
    test
  </div>
</label>
`;
